docs/src/explanations/interfaces-and-connections.md

```diff
@@ -81,10 +81,51 @@ class Block extends Module {
 ```
 where <> bulk connects interfaces of opposite gender between sibling modules or interfaces of the same gender between parent/child modules.
 
-Bulk connections connect leaf ports of the same name to each other. If the names do not match or are missing, Chisel does not generate a connection.
+Bulk connections connect leaf ports of the same name to each other. The Scala types of the Bundles are not required to match. If one named signal is missing from either side, Chisel will give an error such as in the following example:
```

Comment:
```suggestion
The bidirectional connection operator `<>` connects leaf ports of the same name to each other. The Scala types of the Bundles are not required to match. If one named signal is missing from either side, Chisel will give an error such as in the following example:
```

Can you change the name of this section to "## Bidirectional Bulk Connections" ?